tests/PopescuDan.spec.js

```diff
@@ -0,0 +1,9 @@
+// @ts-check
+const { test, expect } = require("@playwright/test");
+
+test("has title", async ({ page }) => {
+  await page.goto("https://manhuaplus.com");
```

Comment:
the website we are testing is https://magento.softwaretestingboard.com/

Reply:
the requirement was to use any website they want. it was a message in discord, you probably didn't see it